runbotics/runbotics-scheduler/src/websocket/bot/process-launch/bot-process-instance.service.ts

```diff
@@ -85,12 +87,33 @@ export class BotProcessService {
     async updateProcessLastRunTime(processInstance: IProcessInstance) {
         if (!processInstance?.process?.id) return;
         const process = await this.processService.findById(processInstance.process.id);
-        
+
         if (process && processInstance.created) {
             await this.processService.partialUpdate({ id: process.id, lastRun: processInstance.created });
         }
     }
 
+    async notifyAboutProcessInstanceStatus(processInstance: IProcessInstance) {
+        const statusList = [
+            ProcessInstanceStatus.INITIALIZING,
+            ProcessInstanceStatus.COMPLETED,
+            ProcessInstanceStatus.TERMINATED,
+            ProcessInstanceStatus.ERRORED,
+        ];
+
+        if (statusList.includes(processInstance.status)) {
+            const processInstanceNotification: ProcessInstanceNotification = {
+                status: processInstance?.status,
+                updated: processInstance?.updated,
```

Comment:
The `updated` field name is unfortunate, it should be more like `finished`

 Imo you should include date of the process start as well